build.gradle

```diff
@@ -10,6 +10,7 @@ group = 'org.springframework.experimental'
 sourceCompatibility = '17'
 
 repositories {
+	mavenLocal()
```

Comment:
We should not have mavenLocal as it makes sure ci build works.

Reply:
Right. I think we can remove it. I needed it before `sbm-support-rewrite` was available remotely. Thx